### ISSUE TITLE
Any plans on i18n?

### ISSUE THREAD

**Issue by m42e** [anonaddy/anonaddy#674]
Hi, 
appreciate your work. 

Have you thought about i18n anonaddy?
I would like to support, but first I want to know if you have plans on doing so and how you would want to do it.

Best

**Comment by willbrowningme** [comment]
Yes that is still on the [roadmap](https://github.com/orgs/anonaddy/projects/1/views/1?pane=issue&itemId=63479156), I just have a number of higher priority things that I'm working on at the moment so cannot give an estimated date for it.

**Comment by realgooseman** [comment]
When time comes and you're looking for a straightforward way to implement it, I would suggest using translation panels like Weblate or Crowdin for faster community contributions instead of contributors having to clone the repo etc..
